db/migrate/20250206202956_remove_show_in_featured_question_report_from_managed_interactive.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveShowInFeaturedQuestionReportFromManagedInteractive < ActiveRecord::Migration[5.2]
+  def up
+    remove_column :managed_interactives, :show_in_featured_question_report
+  end
+  def down
+    add_column :managed_interactives, :show_in_featured_question_report, :boolean, default: true
+  end
+end
```

Comment:
This was added because I missed it in the ["remove show in featured question report" PR](https://github.com/concord-consortium/lara/pull/1221). It was causing Lucy problems with this work, so we added it here. If we really should break it out into a separate PR, though, let me know.